### PR TITLE
Plan for Jakarta Interceptors 2.2 release

### PR DESCRIPTION
**Pull request by starksm64**
## Specification PR template
When creating a specification project release review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [ ] Specification PDF in the form of jakarta-wombat-spec-x.y.pdf
- [ ] Specification HTML in the form of jakarta-wombat-spec-x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [ ] For a Progress Review, that sufficient progress has been made on a Compatible Implementation and TCK, to ensure that the spec is implementable and testable.
- [ ] For a [Release Review](https://www.eclipse.org/projects/handbook/#release-review), a summary that a Compatible Implementation is complete, passes the TCK, and that the TCK includes sufficient coverage of the specification. The TCK users guide MUST include the instructions to run the compatible implementations used to validate the release.
Instructions MAY be by reference.
   - [ ] Updated release record
   - [ ] Generated IP Log
   - [ ] Email to PMC
   - [ ] Start release review by emailing EMO 
- [ ] The URL of the OSSRH staging repository for the api, javadoc:
      <add URL here>
- [ ] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      <add URL here>
- [ ] The URL of the compatibility certification request issue:
      <add URL here>
- [ ] Specification JavaDoc in the wombat/x.y/apidocs directory. 
If desired, an optional second PR can be created to contain just the JavaDoc in the `apidocs` directory.

Note: If any item does not apply, check it and mark N/A below it.

### CODE REVIEW

**Comment by ivargrimstad** [comment]
@jakartaee/eclipsefdn-webdev Can you take a look at why the preview jobs are failing?

**Comment by chrisguindon** [comment]
@ivargrimstad The build is now passing:
https://app.netlify.com/sites/jakartaee-specifications/deploys/649503fb3017de2859cba291


I made the fix on jakarta.ee:
https://github.com/jakartaee/jakarta.ee/commit/0b3d588113e56251bb5a8903d8ba327a9473959a

However, this is only a temporary fix. We will need to update the node.js version and upgrade to our latest front-end framework next week.

//cc @olivergoulet5

**Comment by chrisguindon** [comment]
> Not sure why the GH checks are not green. I will check tomorrow if the state changes.

I sent a support ticket to see how this can be fixed. However, at this point, I believe the main issue is gone.

**Comment by Emily-Jiang** [comment]
I think the build is still broken.

**Comment by chrisguindon** [comment]
> I think the build is still broken.

@Emily-Jiang I just spoke with Netlify. They recommend that someone makes a new commit to the MR which should then update the status.

File: interceptors/2.2/_index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+<!-- Specify the minimum required Java SE version for this specification -->
+**Java SE 11 or higher**
+
+# Details
+
+* [Jakarta Interceptors 2.2 Release Record](https://projects.eclipse.org/projects/ee4j.interceptors/releases/2.2)
+* [Jakarta Interceptors 2.2 Specification Document](./jakarta-interceptors-spec-2.2.pdf) (PDF)
+* [Jakarta Interceptors 2.2 Specification Document](./jakarta-interceptors-spec-2.2.html) (HTML)
+* [Jakarta Interceptors 2.2 Javadoc](./apidocs)
+* [Jakarta Contexts Dependency Injection 4.0 TCK](https://download.eclipse.org/ee4j/cdi/4.0/cdi-tck-4.0.0-dist.zip)
```
[the remark this answers is not in the document]
Comment (starksm64) [reply interceptors/2.2/_index.md]: Fixed

**Pull request merged**